compiler/rustc_expand/src/mbe/macro_parser.rs

```diff
@@ -463,7 +464,7 @@ impl TtParser {
         // Matcher positions that would be valid if the macro invocation was over now. Only
         // modified if `token == Eof`.
         let mut eof_mps = EofMatcherPositions::None;
-
+        let mut pre_doc_token_idx = None;
```

Comment:
Nit: I think prev would be a better name here, as `pre-` can also indicate something before something else.

Reply:
Forgive my poor English 😂